src/rust/src/lib.rs

```diff
@@ -7,12 +7,15 @@
 #![allow(non_camel_case_types)]
 #![allow(non_snake_case)]
 
+extern crate palette;

```

Comment:
No need of extern crate in Rust 2018, https://doc.rust-lang.org/edition-guide/rust-2018/path-changes.html#no-more-extern-crate